### PR TITLE
fixed user account drop down, removed unusec wallet component, move u…

### PR DESCRIPTION
**Pull request by KatarzynaStefanowiczPagepro**
…ser info avatar to separate component

## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

### CODE REVIEW

**Comment by KatarzynaStefanowiczPagepro** [comment]
https://tw.auroracreation.com/app/tasks/15516473

Fixed the user account drop-down,
Removed unused wallet component
Moved the user info avatar to separate, more reusable component 

image

File: src/components/common/Extensions/UserNavigation/partials/UserMenu/UserMenu.tsx [hunk 1]
```diff
@@ -109,61 +108,51 @@ const UserMenu: FC<UserMenuProps> = ({
...
+              <li key={id} className="mb-2 last:mb-0 sm:mb-0">
+                {link ? (
+                  <Link to={link} className="navigation-link">
+                    <Icon name={icon} appearance={{ size: 'tiny' }} />
+                    <p className="ml-2">{formatMessage({ id: itemName })}</p>
+                  </Link>
+                ) : (
+                  <button
+                    type="button"
+                    className="navigation-link"
```
Comment (krawczykkatarzyna) [inline src/components/common/Extensions/UserNavigation/partials/UserMenu/UserMenu.tsx:131]: aria-label is missing

File: src/components/common/Extensions/UserNavigation/partials/UserSubmenu/UserSubmenu.tsx [hunk 2]
```diff
@@ -12,8 +12,14 @@ const UserSubmenu: FC<UserSubmenuProps> = ({ submenuId }) => (
...
         <NavLink to={url} className="flex items-center">
-          <Icon name={icon} appearance={{ size: 'extraSmall' }} />
-          <p className="ml-2 heading-5 sm:font-normal sm:text-md">{label}</p>
+          <span className="flex items-center shrink-0 mr-2 sm:mr-0 flex-grow">
+            <Icon
+              name={icon}
+              appearance={{ size: 'small' }}
+              className="navigation-link"
+            />
+            <p className="ml-2 ">{label}</p>{' '}
```
Comment (krawczykkatarzyna) [inline src/components/common/Extensions/UserNavigation/partials/UserSubmenu/UserSubmenu.tsx:21]: {' '} is it needed?

Comment (KatarzynaStefanowiczPagepro) [reply src/components/common/Extensions/UserNavigation/partials/UserSubmenu/UserSubmenu.tsx:21]: Missclick :)

File: src/components/common/Extensions/UserNavigation/partials/UserMenu/UserMenu.tsx (hunk 1 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/common/Extensions/UserNavigation/partials/UserMenu/UserMenu.tsx:131]: aria-label isn't required when inside the button is text, here is a item name

**Review by CzarekDryl — APPROVED** [review]

**Review by krawczykkatarzyna — APPROVED** [review]

**Pull request merged**